test/TestBytesLib1.sol

```diff
@@ -481,160 +556,157 @@ contract TestBytesLib1 is Test {
     * Equality Non-aligned Tests
     */
 
-    /// @dev this test passes with truffle.
-    // function testEqualNonAligned4Bytes() public {
```

Comment:
Why are we deleting the non-aligned ones?

Reply:
I did this in the original PR when I attempted to replace all equality checks with foundry checks.

You can see these tests are added back [here](https://github.com/GNSPS/solidity-bytes-utils/pull/65/commits/93043815b9dd91bd7f64d563e97ff32f12a69502#diff-a22abc84191afcf21161cbd24f542b66e4e6fc9dd25c656a3d8c10883e8fb78aR562).